resources/lang/nl/matches.php

```diff
@@ -8,7 +8,7 @@
         'beatmap-deleted' => 'verwijderde beatmap',
         'difference' => 'met :difference',
         'failed' => 'GEFAALD',
-        'header' => 'Multiplayer Spellen',
+        'header' => 'Multiplayer Wedstrijden',
```

Comment:
Is this part of multi games? 'Wedstrijden' stands for competitive games. If it's casual, 'Spellen' fits better